docs/opencost-exporter.md

```diff
@@ -0,0 +1,120 @@
+
+## Installing
+
+> Note: all deployments of Kubecost function as a Prometheus metric exporter. We strongly recommend helm as an install path to take advantage of Kubecost’s full potential. [View recommended install](http://docs.kubecost.com/install).
+
+If you would prefer to not use the recommended install option and just deploy the OpenCost open source cost model as a metric exporter, you can follow these steps:
+
+1. This will target a Prometheus Server named my-prometheus-server in the prom namesapace. If yours is somewhere else, you will need to modify accordingly. See `Prerequisites` in [basic-setup.md](./basic-setup.md)
+
+1. Apply the combined YAML:
+
+```
+kubectl apply -n opencost -f https://raw.githubusercontent.com/kubecost/opencost/develop/kubernetes/exporter/exporter.yaml
+```
+
+1. To verify that metrics are available:
+
+    ```
+    kubectl port-forward --namespace opencost service/opencost 9003
+    ```
+
+    Visit [http://localhost:9003/metrics](http://localhost:9003/metrics) to see exported metrics
+
+Add Kubecost scrape config to Prom ([more info](https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus))
+
+```yaml
+- job_name: opencost
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+    - targets: ['opencost.opencost.:9003']
+```
+
+Done! Kubecost is now exporting cost metrics. See the following sections for different metrics available and query examples.
+
+## Available Prometheus Metrics
+
+| Metric       | Description                                                                                            |
+| ------------ | ------------------------------------------------------------------------------------------------------ |
+| node_cpu_hourly_cost | Hourly cost per vCPU on this node  |
+| node_gpu_hourly_cost | Hourly cost per GPU on this node  |
+| node_ram_hourly_cost   | Hourly cost per Gb of memory on this node                       |
+| node_total_hourly_cost   | Total node cost per hour                       |
+| kubecost_load_balancer_cost   | Hourly cost of a load balancer                 |
+| kubecost_cluster_management_cost | Hourly management fee per cluster                 |
+| container_cpu_allocation   | Average number of CPUs requested over last 1m                      |
+| container_memory_allocation_bytes   | Average bytes of RAM requested over last 1m                 |
+
+By default, all cost metrics are based on public billing APIs. See the Limitations section below about reflecting your precise billing information. Supported platforms are AWS, Azure, and GCP. For on-prem clusters, prices are based on configurable defaults.
+
+More metrics are available in the recommended install path and are described in [PROMETHEUS.md](PROMETHEUS.md).
```

Comment:
I believe this should link to https://github.com/kubecost/opencost/blob/master/PROMETHEUS.md

Reply:
or do we move the whole doc to the website?
because this is a good doc for the website, and it would follow the practice for what we're doing here.

Reply:
meaning, move this: https://github.com/kubecost/opencost/blob/master/PROMETHEUS.md
to opencost-website

Reply:
Could move now or could do in another PR. I think either works, but agree would be nice to have here. 

The reason why it may make sense to do in a second PR is that deleting that file may be tricky to make sure we aren't breaking other refs.

Reply:
added file back with a note that it moved to the docs website.

Reply:
Vercel is still erroring on this link. Might be because of the file name being lowercase but the link being uppercase?